va/Android.mk

```diff
@@ -108,6 +109,7 @@ LOCAL_CFLAGS += \
 
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libva-egl
+LOCAL_PROPRIETARY_MODULE := true
```

Comment:
libva provides a new mechanism to share buffer between libva and the 3rd library, are you sure this library still used on Android?